public/assets/js/spongedownloads.js

```diff
@@ -0,0 +1,388 @@
+const API = "https://dev.minecrell.net/downloads"
```

Comment:
@jamierocks's suggestion is probably the better one but if we do that we should stop commiting the compiled JS/CSS files and just require to run gulp to setup the project.